data.md

```diff
@@ -244,6 +251,14 @@ Sehi L'Yi, Harrison G Zhang, Andrew P Mar, Thomas C Smits, Lawrence Weru, Sofía
     outline: 3px solid #005fcc;
   }
 
+  .file-title {
+    font-weight: bold;
```

Comment:
I am not sure about this. I think with this, there is a lot of bolded information, such that a user can't directly see what is important. This way, the expand-triangle and download-arrow are also less clear. I do think it's good to visually make it more important than the file name, but perhaps we can find a different way? Make the file name italics?

![image](https://github.com/user-attachments/assets/1b5ddfd2-e895-4220-a4af-ed4a82ab29fb)

Reply:
I realized file names are not the kind of details that most users want to see. I moved the file names under the menu (i.e., above the preview table).

![Screenshot 2024-12-10 at 2 43 21 PM](https://github.com/user-attachments/assets/58f00c34-4b55-46cc-95a3-b9764cc84521)